eng/scripts/ChangedFiles-Functions.ps1

```diff
@@ -0,0 +1,56 @@
+function Get-ChangedFiles($baseCommitsh="HEAD^", $targetCommitish="HEAD", $diffFilter="d")
+{
+    # diff-filter=d is to exclude any deleted files as we don't generally 
```

Comment:
Check "TypeSpec Validation" does want to use deleted files as a signal.  If any file is deleted under folder `specification/foo`, we want to validate all TypeSpec specs under `specification/foo`, to ensure the deletion doesn't break anything.

As long as "TypeSpec Validation" can enable this by overriding the default `diff-filter`, we can make it work.  But I wonder if excluding deleted files is the correct default?

Reply:
In general, for most things that are taking action on the list of files they don't want files that no longer exist because they cannot read them.

For tsv what happens in a case when an entire typspec project gets deleted? I guess you simply won't find a tspconfig file for those and it will be a noop so that should be OK for your scenario.